### PR TITLE
Ubuntu/mantic

### PR DESCRIPTION
**Pull request by blackboxsw**
Add missing quilt patch for 24.2 release. mantic original cloud-init original release in mantic 23.3.

### CODE REVIEW

File: debian/patches/deprecation-version-boundary.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
+--- a/cloudinit/features.py
++++ b/cloudinit/features.py
+@@ -87,7 +87,7 @@ On Debian and Ubuntu systems, cc_apt_con
+ to write /etc/apt/sources.list directly.
+ """
+ 
+-DEPRECATION_INFO_BOUNDARY = "devel"
++DEPRECATION_INFO_BOUNDARY = "24.1"
```
Comment (TheRealFalcon) [inline debian/patches/deprecation-version-boundary.patch:17]:
```suggestion
+DEPRECATION_INFO_BOUNDARY = "23.3"
```

**Review by TheRealFalcon — CHANGES_REQUESTED** [review]
mantic contains 23.3.1-0ubuntu2

File: debian/patches/deprecation-version-boundary.patch (hunk 1 above)
Comment (blackboxsw) [reply debian/patches/deprecation-version-boundary.patch:17]: oops was looking at mantic-updates in rmadison  `cloud-init | 23.3.1-0ubuntu2                    | mantic          | source, all`

**Review by TheRealFalcon — APPROVED** [review]
LGTM!

**Pull request merged**